ql/src/java/org/apache/hadoop/hive/ql/exec/DDLTask.java

```diff
@@ -3597,6 +3604,14 @@ static StringBuilder appendNonNull(StringBuilder builder, Object value, boolean
    *           Throws this exception if an unexpected error occurs.
    */
   private int alterTable(Hive db, AlterTableDesc alterTbl) throws HiveException {
+    if (alterTbl.getOp() == AlterTableDesc.AlterTableTypes.RENAME) {
+      String names[] = Utilities.getDbTableName(alterTbl.getOldName());
+      if (Utils.isBootstrapDumpInProgress(db, names[0])) {
```

Comment:
Using the database name at index 0 might not be correct, as the same table might be in different database with replication happening on a different database

Reply:
Did you mean the case where table is renamed/moved to another database?
In this case, it is not at all a problem even if we allow rename as we currently don't support replication for operations across database. So, it is enough to check if dump progress for source database alone.